scripts/segmentation_template.md

```diff
@@ -293,3 +293,41 @@ yaml.dump(to_yaml(platform, flight_id, ds, segments, events),
           open(f"../flight_segment_files/{flight_id}.yaml", "w"),
           sort_keys=False)
 ```
+
+## Import YAML and test it
+
+```python
+flight = yaml.safe_load(open(f"../flight_segment_files/{flight_id}.yaml", "r"))
+```
+
+```python
+kinds = set(k for s in segments for k in s["kinds"])
+```
+
+```python
+fig, ax = plt.subplots()
+
+for k, c in zip(['straight_leg', 'circle', ], ["C0", "C1"]):
+    for s in flight["segments"]:
+        if k in s["kinds"]:
+            t = slice(s["start"], s["end"])
+            ax.plot(ds.lon.sel(time=t), ds.lat.sel(time=t), c=c, label=s["name"])
+ax.set_xlabel("longitude / °")
+ax.set_ylabel("latitude / °");
```

Comment:
Is the semicolon at the end of this line on purpose? Apart from this, everything looks good and PR can be merged 👍🏻

Reply:
In notebooks, the last statement in a cell is treated as the return value of that cell and is usually displayed. I assume, the semicolon is used in this case, to make the empty space after the semicolon the last thing, which effectively means returning `None` from the cell.
As `None` is not displayed when returning from a cell, people tend to use `;` at the end of a cell as a shorthand to silence otherwise printed output (in this case the somewhat random object which would be returned by the `set_ylabel` function).